python/kvikio/kvikio/_lib/remote_handle.pyx

```diff
@@ -23,6 +23,15 @@ cdef extern from "<kvikio/remote_handle.hpp>" nogil:
     cdef cppclass cpp_HttpEndpoint "kvikio::HttpEndpoint":
         cpp_HttpEndpoint(string url) except +
 
+    cdef cppclass cpp_S3Endpoint "kvikio::S3Endpoint":
```

Comment:
It's not strictly in scope for this PR, but why not add a namespace clause to the `cdef extern` on line 19 so that you don't have to manually alias each of these types? You can also drop the `cpp_` prefixes then.

Reply:
Heh, @wence- asked the same thing. I just like the `cpp_` so I don't mix up what is C++ and what is Cython/Python. This also makes it easier to search the codebase.

Reply:
Fair enough, whatever you prefer is fine.